CHANGELOG.md

```diff
@@ -97,7 +104,17 @@ of response and not the raw API endpoint response
   - **CUMULUS-2634**
     - Changed `sfEventSqsToDbRecords` Lambda to use new upsert helpers for executions, granules, and PDRs
     to ensure out-of-order writes are handled correctly when writing to Elasticsearch
-    
+  - **CUMULUS-2510**
+    - Removed `stream_enabled` and `stream_view_type` from `executions_table` TF
+      definition.
+    - Removed `aws_lambda_event_source_mapping` TF definition on executions DynamoDB table.
```

Comment:
These should go under a `Removed` header

Reply:
https://github.com/nasa/cumulus/pull/2422/commits/bd0112b50cc0fda564ec0be0a84a830659daf1ea